cinema/models.py

```diff
@@ -122,8 +120,8 @@ def save(
         )
 
     def __str__(self):
-        return (f"{str(self.movie_session)} "
-                f"(row: {self.row}, seat: {self.seat})")
+        return f"{str(self.movie_session)} "\
+               f"(row: {self.row}, seat: {self.seat})"
```

Comment:
There is a syntax error in the `__str__` method of the `Ticket` class. The line continuation character `\` should be placed at the end of line 123, not at the beginning of line 124. This will fix the syntax error.